### DISCUSSION TITLE
Question: Is there something "equivalent" to pmtiles, compressed single-file tile archives, for Reverse Geocoding?

### DISCUSSION THREAD

**Discussion by raphael10-collab** [protomaps/PMTiles#437] [Ideas]
Is there something "equivalent" to pmtiles, Cloud-optimized + compressed single-file tile archives for vector and raster maps,  for Reverse Geocoding, which could effectively and lightly replace the heavy Nominatim Server?

**Comment by nvkelso** [comment]
See https://millsfield.sfomuseum.org/blog/2022/12/19/pmtiles-pip/

**Comment by do-me** [comment]
@raphael10-collab Yes, there is https://airmail.rs/. I think it's exactly what you're looking for!
